pkg/reconciler/events/k8sevent/events.go

```diff
@@ -20,14 +20,14 @@ import (
 	"regexp"
 	"testing"
 	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 // CheckEventsOrdered checks that the events received via the given chan are the same as wantEvents,
 // in the same order.
 func CheckEventsOrdered(t *testing.T, eventChan chan string, testName string, wantEvents []string) error {
```

Comment:
Wondering why this function is not part of `_test.go` when its written to be used for only **tests**? 🤔 This way it can be consumed outside of `tests`.  Shall we make the wait interval configurable such that the user is responsible for choosing the appropriate wait time?

Reply:
> Wondering why this function is not part of `_test.go` when its written to be used for only **tests**? 🤔 This way it can be consumed outside of `tests`. Shall we make the wait interval configurable such that the user is responsible for choosing the appropriate wait time?

It's because the functions in `_test.go` cannot be used in other pkgs